views/partials/home/simple.mako

```diff
@@ -15,8 +15,10 @@
 % for cur_show_list in show_lists:
     <% cur_list_type = cur_show_list[0] %>
     <% my_show_list = list(cur_show_list[1]) %>
-    % if cur_list_type == "Anime":
-        <h1 class="header">Anime List</h1>
+    % if len(show_lists) > 1:
+        % if len(show_lists) > 1:
```

Comment:
And here

Reply:
Sorry for that, copy paste error... :roll_eyes: